tests/ui/test_ui.py

```diff
@@ -248,11 +248,37 @@ def test_keypress_ALL_MENTIONS(self, view, mocker, key, widget_size):
         view.body.focus_col = None
         view.controller.is_in_editor_mode = lambda: False
         size = widget_size(view)
-        view.model.controller.narrow_to_all_mentions = mocker.Mock()
+        view.controller.narrow_to_all_mentions = mocker.Mock()
 
         view.keypress(size, key)
 
-        view.model.controller.narrow_to_all_mentions.assert_called_once_with()
+        view.controller.narrow_to_all_mentions.assert_called_once_with()
+        assert view.body.focus_col == 1
+
+    @pytest.mark.parametrize("key", keys_for_command("ALL_PM"))
+    def test_keypress_ALL_PM(self, view, mocker, key, widget_size):
```

Comment:
I did try that, a lot, but it wasn't working maybe because patch did not want to take a variable containing a function as its argument? Maybe it was patching the variable itself? I wasn't able to make anything work so I went with 3 copies.

Reply:
Without seeing the different approach it's difficult to comment too much. Patch normally takes a string, so it should be able to parametrize over?